code/obj/item/device/nuclear_op_items.dm

```diff
@@ -0,0 +1,919 @@
+						if(confirmation == "Yes")
+							var/confirmation2 = input(user, "Are you EXTREMELY sure? There's no coming back!") in list("Yes", "No")
+							if(confirmation2 == "Yes")
+								send_to_pod(user)
+							else
+								return
+						else
+							return
+					else
+						var/confirmation = input(user, "Are you sure you would like to deploy? You have [length(chosen_mobs)] who will deploy with you.") in list("Yes", "No")
+						if(confirmation == "Yes")
+							var/confirmation2 = input(user, "Are you EXTREMELY sure? There's no coming back!") in list("Yes", "No")
+							if(confirmation2 == "Yes")
+								send_to_pod(user)
+							else
+								return
+						else
+							return
+				if("Cancel")
+					return
+
+
+	proc/choose_area(var/mob/user)
+		var/temp_people_count = 10 //sanity check to make sure there's enough turfs to land on
+		var/list/area/filtered_areas = get_teleareas()
+		var/list/turf/check_turfs = list()
+		for(var/mob/living/carbon/M in range(4, user.loc))
+			temp_people_count += 1
+		for(var/area/A in filtered_areas)
+			for(var/turf/T in get_area_turfs(A, TRUE))
+				check_turfs += T
+			if(!(length(check_turfs) >= temp_people_count))
+				filtered_areas -= A
+				continue
+		var/area/temp_area = input("Choose Landing Area") as null|anything in filtered_areas
+		src.landing_area = get_telearea(temp_area)
+		if (!src.landing_area)
+			return FALSE
+		var/list/turf/possible_turfs = list()
+		for(var/turf/T in get_area_turfs(src.landing_area, TRUE))
+			possible_turfs += T
+
+	proc/send_to_pod(var/mob/user)
+		for(var/mob/living/carbon/M in range(4, user.loc))
+			SPAWN_DBG(0)
+				var/L = pick_landmark(LANDMARK_SYNDICATE_ASSAULT_POD_TELE)
+				if(!L) //fuck
+					return
+				for(var/obj/item/remote/syndicate_teleporter/T in M.get_all_items_on_mob())
+					qdel(T) //Emphasizing that there really is no easy way back if you go this way
+				playsound(M, "sound/effects/teleport.ogg", 30, 1)
+				var/obj/decal/teleport_swirl/S = new/obj/decal/teleport_swirl(M)
+				M.set_loc(L)
+				var/obj/decal/residual_energy/R = new/obj/decal/residual_energy(L)
+				playsound(L, "sound/effects/teleport.ogg", 30, 1)
+				SPAWN_DBG(1 SECOND)
+					qdel(S)
+					qdel(R)
+			sent_mobs += M
+		for(var/obj/machinery/nuclearbomb/NB in range(4, user.loc))
+			SPAWN_DBG(0)
+				var/L = pick_landmark(LANDMARK_SYNDICATE_ASSAULT_POD_TELE)
+				if(!L)
+					return
+				playsound(NB, "sound/effects/teleport.ogg", 30, 1)
+				var/obj/decal/teleport_swirl/S = new/obj/decal/teleport_swirl(NB)
+				NB.set_loc(L)
+				var/obj/decal/residual_energy/R = new/obj/decal/residual_energy(L)
+				playsound(L, "sound/effects/teleport.ogg", 30, 1)
+				SPAWN_DBG(1 SECOND)
+					qdel(S)
+					qdel(R)
+			nuclear_bombs += NB
+		src.used = TRUE
+		for(var/obj/machinery/computer/security/pod_timer/S in range(1, pick_landmark(LANDMARK_SYNDICATE_ASSAULT_POD_COMP))) //This is the only way I could make this work
```

Comment:
what? this doesn't make sense

Reply:
I tried creating a `new` computer on the landmark and editing its maptext, but that didn't work, I eventually landed on this

Reply:
no like `range(1 to THING)` makes no sense - does it use its loc or something weird??

Reply:
`range` takes two arguments, distance and starting location, it checks all turfs within 1 tile (so, on the tile) for a pod_timer computer

Reply:
fuck i was thinking in rust